docs/background.md

```diff
@@ -0,0 +1,89 @@
+# Background
+
+## Explosion of 'AI Model Repositories'
+
+A list that starts off incomplete, and will most likely always be a subset, of places where "AI Model Metadata" is 
+accessible in some form or fashion:
+- HuggingFace
+- Ollama
+- Kubeflow Model Registry
+- KServe CRDs in K8s clusters
+- MLFlow Model Registry
+- OCI image registries like quay.io, registry.redhat.io, or docker.io
+- API GWs like 3Scale or Kong
+
+Each have some form of REST API.  Most if not all have a CLI that interacts with said REST API.
+
+Building some form of normalization around taking data from those sources and constructing Backstage catalog
+artifacts or "Entities" (where, by the way, the Backstage Catalog also has a REST API).
+
+## Developer exploration vs. prescription from the enterprise
+
+So "who" should be making the decision of which AI Models land in a Backstage instance to facilitate the use of AI
+in applications developed with Backstage?
+
+Whether the answer includes
+- any developer on the given Backstage instance (where the developer team may have set it up)
+- select developers on the instance
+- or only the DevOps or MLOps or Platform engineers who set up the Backstage instances, where those folks are different people than the developers using Backstage 
+
+can potentially result in different preference for "how" the Catalog is updated.
+
+## Personas and their scenarios
+
+Boiling that down into an agile story description:
+
+As a platform/MLOPs engineer, I want to administrate the Backstage Catalog from the command line so that I can better automate administration of Backstage for AI related application development.
+
+As a developer/tester engineer, I want to administrate the Backstage Catalog from the command line so that I can better automate verification pipelines for testing of Backstage’s AI related features.
```

Comment:
Nit: `developer/tester engineer` reads weird to me personally

Reply:
ok I will attempt to reword .... any suggestions on how I should capture both dev and QE ?

Reply:
We could probably swap it to 
```suggestion
As a development/test engineer, I want to administrate the Backstage Catalog from the command line so that I can better automate verification pipelines for testing of Backstage’s AI related features.
```

Or maybe
```suggestion
As a developer/tester, I want to administrate the Backstage Catalog from the command line so that I can better automate verification pipelines for testing of Backstage’s AI related features.
```

Reply:
I think I'm going to go with DevOps